safe/impact_functions/ash/ash_raster_landcover/impact_function.py

```diff
@@ -158,6 +159,16 @@ def run(self):
         del writer
         impact_layer = QgsVectorLayer(filename, 'Impacted Land Cover', 'ogr')
 
+        # find unaffected features
+        unaffected_feats = []
+        target_field_index = impact_layer.fieldNameIndex(self.target_field)
+        for f in impact_layer.getFeatures():
+            haz_class = f.attributes()[target_field_index]
+            if haz_class == self.hazard_classes[0]:
+                unaffected_feats.append(f.id())
+
+        impact_layer.dataProvider().deleteFeatures(unaffected_feats)
```

Comment:
Fix this to delete unnaffected features from the layer. It shouldn't be included. Unaffected landcover should be excluded, since it doesn't make sense to include it if it is unaffected.